packages/api/tests/models/rules/test-rules-model.js

```diff
@@ -196,6 +196,19 @@ test.serial('Creating an invalid rule does not create workflow triggers', async
   }
 });
 
+test('Creating a rule with a createdAt and updatedAt date assigns the correct values to createdAt and provides a new updatedAt', async (t) => {
+  const newRule = fakeRuleFactoryV2();
+  const currentDate = new Date('1/2/2030').getTime();
```

Comment:
No real objection, but why use this future date?

Reply:
For fun 😮 If that's not a sufficient reason, I can change it do a date you think is more reasonable.